src/components/Testimonials.tsx

```diff
@@ -7,21 +7,21 @@ const testimonialData = [
     description:
       'Consisting of core Ethereum developers, Protocol Guild prioritizes minimal governance in member engagement.',
     buttonText: 'Read More',
-    href: 'https://summon.daohaus.fun/',
+    href: 'https://summon.daohaus.club/',
   },
   {
     title: 'Gitcoin FDD',
     description:
       'As a sub-group within Gitcoin, the Fraud Detection team allows for active and inactive status that adjusts individual voting power.',
     buttonText: 'Read More',
-    href: 'https://admin.daohaus.fun/',
+    href: 'https://admin.daohaus.club/',
   },
   {
     title: 'Raid Brood',
     description:
       'NFTs are rewards for purchasing beer and then staked for membership into a DAO that collectively owns products.',
     buttonText: 'Read More',
-    href: 'https://hub.daohaus.fun/',
+    href: 'https://hub.daohaus.club/',
```

Comment:
This is a dead link. Probably some placeholder?

Maybe use https://brood.raidguild.org/